src/test/java/net/imglib2/view/ConcatenateViewTest.java

```diff
@@ -57,19 +64,60 @@
 
 	private final long[] dim = new long[] { 3, 4, 5, 6 };
 
+	private final long[] revertedDim = new long[] { 6, 5, 4, 3 };
```

Comment:
There is no reason to have `dim`, `revertedDim`, `divider`, `axis` and `revertedAxis` as fields. They should be declared as variables in the methods testConcatenate and testConcatenateFirstAxis.

The prefix `reverted` is kind of confusing. Because it rises the idea that some reverting happens. But that's actually not the case. Ok the order of the numbers are reverted, but that has no meaning for the test.

Reply:
Addresed in f7d1841, passing values as anonymous variables now.